packages/documentsearch/src/searchview.tsx

```diff
@@ -270,15 +272,34 @@ interface IUpDownProps {
   trans: TranslationBundle;
 }
 
+
+
+
+// Getting the CommandRegistry object
+const commands = new CommandRegistry();
+//commands.addCommand(...);
+
+// Getting the key bindings for the next and previous commands
```

Comment:
The code below is ok but should be in the `UpDownButtons` component as the keyboard shortcuts may be changed by the user.